src/main/resources/templates/jenkins/c/Jenkinsfile

```diff
@@ -78,8 +83,8 @@ pipeline {
                             cd tests
 
                             python3 Tests.py s
-                            rm Tests.py
-                            rm -rf ./tests
+                            #rm Tests.py
```

Comment:
Why do you comment out those commands?

Reply:
Good catch. I was testing something a forgot to remove the comment